packages/map-template/src/components/LocationDetails/LocationDetails.scss

```diff
@@ -129,5 +152,13 @@
     &__content {
         display: grid;
         gap: var(--spacing-xx-small);
+
+        mi-location-info {
+            overflow: hidden;
+            display: -webkit-box;
+            -webkit-line-clamp: 3;
+            line-clamp: 3;
+            -webkit-box-orient: vertical;
```

Comment:
https://caniuse.com/mdn-css_properties_box-orient

"This feature is deprecated/obsolete and should not be used."

https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient

"Do not use it on production sites facing the Web"

Maybe we should reconsider...

Reply:
UX decided to allow the location info to take as much space as needed. We will look more into it depending on the feedback that we get from the customers, but for now we'll not add any length/height restrictions.